### ISSUE TITLE
SyntaxError on import for python 3.6

### ISSUE THREAD

**Issue by mauvilsa** [mdomke/schwifty#59]
```bash
$ virtualenv -p python3.6 venv36
...
$ . venv36/bin/activate
$ pip3 install schwifty
...
Successfully installed iso3166-2.0.2 pycountry-20.7.3 schwifty-2021.6.1
$ python3 -c "import schwifty"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/venv36/lib/python3.6/site-packages/schwifty/__init__.py", line 3, in <module>
    from schwifty.bic import BIC
  File "/tmp/venv36/lib/python3.6/site-packages/schwifty/bic.py", line 1
    from __future__ import annotations
    ^
SyntaxError: future feature annotations is not defined
```

Has support for python 3.6 been dropped? The package metadata does not prevent installing in python 3.6. If support has been dropped then this should be specified in `python_requires`. If it hasn't been dropped then this is a bug.

**Comment by ckoehn** [comment]
Python 3.6 support has been removed in [2021.04.0](https://github.com/mdomke/schwifty/releases/tag/2021.04.0).

**Comment by mdomke** [comment]
This has been addressed in the latest release [2021.10.0](https://github.com/mdomke/schwifty/releases/tag/2021.10.0). The `setup.cfg` now uses `option.python_requires` as suggested. Thanks for the hint!
